<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>股票行情监控</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="searchContainer">
      <item>
       <layout class="QHBoxLayout" name="searchLayout">
        <item>
         <widget class="QLineEdit" name="stockCodeEdit">
          <property name="placeholderText">
           <string>输入股票代码（如：sh600000）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="text">
           <string>实时数据</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="historyLayout">
        <item>
         <widget class="QDateTimeEdit" name="startDateEdit">
          <property name="displayFormat">
           <string>yyyy-MM-dd HH:mm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dateRangeLabel">
          <property name="text">
           <string>至</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="endDateEdit">
          <property name="displayFormat">
           <string>yyyy-MM-dd HH:mm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="historyButton">
          <property name="text">
           <string>历史数据</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="infoLayout">
      <item>
       <widget class="QLabel" name="stockNameLabel">
        <property name="text">
         <string>股票名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentPriceLabel">
        <property name="text">
         <string>当前价格：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QChartView" name="chartView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>